libraries/classes/Controllers/Table/ReplaceController.php

```diff
@@ -664,7 +664,7 @@ public function index(): void
         if ($goto_include === '/database/sql') {
             /** @var DatabaseSqlController $controller */
             $controller = $containerBuilder->get(DatabaseSqlController::class);
```

Comment:
```suggestion
            $containerBuilder->get(DatabaseSqlController::class)();
```
This could be an alternative, but is it a good one ?. I will let you decide ^^

Reply:
I also prefer this way. :)